### ISSUE TITLE
Module installed on PrestaShop 1.7.6.1 : in demo 404 - File or directory not found.

### ISSUE THREAD

**Issue by KInAlves** [VivaPayments/API#1185]
Hello
We have install the presyashop module recover on your servers. We had several problems. The first problem was solved using github.com, it concerned the translations. The second problem is on the demonstration environment, where we have an error:
Server Error
404 - File or directory not found.
The resource you are looking for may have been removed, or it's changed, or is temporarily unavailable.

We have found solutions but have no effect on our prestashop. We attach the screenshots.
Thanks
![screenshot2](https://user-images.githubusercontent.com/57185113/67868489-597ce180-fb2c-11e9-98a5-f5c770a39a2d.jpg)
![screenshot1](https://user-images.githubusercontent.com/57185113/67868493-5a157800-fb2c-11e9-9a4f-dd87f645dea9.jpg)
![screenshot3](https://user-images.githubusercontent.com/57185113/67868494-5aae0e80-fb2c-11e9-8be1-d7f3712897c2.jpg)

**Comment by VivaWalletSupport** [comment]
Hello, as you seem to use the credentials from your live Viva account, you would have to use the production URL in your module settings. Otherise setup an account at https://demo.vivapayments.com and use those credentials (MerchantID, API Key, Source Code) in your modules settings.

**Comment by KInAlves** [comment]
Hello
It is impossible to create an account on the demo space. Test on Firefox, Chrome and Safari without success. It's complicated to test under these conditions.

**Comment by KInAlves** [comment]
So impossible to create an account on the demo space. So in the end, we went back to the production area, we created a 1 € product on our prestashop shop. And then carry out real tests with our credit cards. We had no problem with traditional bank cards. By cons we tested Nickel mastercard cards on which does not appear the name of the owner of the card and there we have a payment error, we have to test two fosi with the same card, the first time we received the code 3DS but the payment failed, the second time we were redirected directly to the store after having validate the card without even receiving the code 3DS.
We take the risk of launching the store in production with the concern of Nickel Mastercard but we can not afford to stay longer without payment on our site which has been in maintenance for more than 10 days.
Thank you for your comeback.
